src/Broadway/EventStore/DBALEventStore.php

```diff
@@ -69,10 +72,41 @@ public function __construct(
     /**
      * {@inheritDoc}
      */
-    public function load($id)
+    public function load($id, $playhead = 0)
     {
         $statement = $this->prepareLoadStatement();
         $statement->bindValue('uuid', $this->convertIdentifierToStorageValue($id));
+        $statement->bindValue('playhead', $playhead);
+        $statement->execute();
+
+        $events = $this->executeStatement($statement);
+
+        if (empty($events)) {
+            throw new EventStreamNotFoundException(sprintf('EventStream not found for aggregate with id %s', $id));
+        }
```

Comment:
Oh jeez you're right. I guess it should only be thrown when you're trying to load *all* events of an aggregate (ie from playhead 0).